PULL_REQUEST_TEMPLATE

```diff
@@ -21,11 +21,4 @@ Add a description of your changes here.
 
 ### Reviewers
 
-If you've been tagged for review, please "Start a review" with the GitHub GUI.
-
-- Code review: @Reviewer1, @Reviewer2
-- UI strings/error msgs review: @...
-- UX review: @...
-- Accessibility review: @...
-- Product review: @...
-- FYI @someone
+Add reviewers with the github GUI. 
```

Comment:
I would suggest removing the "Reviewers" section entirely. We should already know how to request reviews, or maybe this can be added to the README if needed (although I don't think it is).